src/components/JdsCard/Card.stories.js

```diff
@@ -0,0 +1,120 @@
+import JdsCard from './Card.vue';
+import storybookMixin from '../../utils/storybook'
+
+export default {
+  component: JdsCard,
+  title: 'Components/Card',
+  parameters: {
+    backgrounds: {
+      default: 'gray'
+    },
+  },
+  argTypes: {
+    className: {
+      name: 'class',
+      defaultValue: 'bg-white',
+      control: {
+        type: 'text'
+      }
+    },
+    width: {
+      name: 'style.width',
+      defaultValue: '355px',
+      description: 'Card width in pixel',
+      control: {
+        type: 'text'
+      }
+    },
+    hideBody: {
+      defaultValue: false,
+      table: {
+        disable: true
+      }
+    }
+  }
+};
+
+const Template = (args, context) => {
+  return {
+    name: 'JdsCardStories',
+    components: { JdsCard },
+    mixins: [storybookMixin(args, context)],
+    props: ['className', 'width', 'hideBody'],
+    template: `
+      <jds-card
+        :class="$props.className"
+        :style="{ width: $props.width }"
+        v-bind="$props"
+        v-on="events"
+      >
+        <div
+          v-if="!hideBody"
+          class="font-sans-1 text-gray-800"
+        >
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+          Non aliquam nunc etiam risus consequat vestibulum tellus eget orci.
+        </div>
```

Comment:
Hi! I'm just wondering, could I change this text on the Storybook later? @adrianpdm

Reply:
thank you, fixed in [this commit](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/56/commits/b5075f205c8e36ac5ee0b263974776638a9201af)